### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by chrisnegus**
**Description of changes:**

Updated the README.md file with new descriptive text. I ended up merging some of the existing text with text from the introduction in the docs. @ellistarn Let me know if you had something different in mind.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,30 @@
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/awslabs/karpenter/issues)
 
 ![](website/static/banner.png)
+{{% alert title="Note" color="primary" %}}
```
Comment (geoffcline) [inline README.md:8]: I didn't know GitHub markdown supported this!

File: README.md [hunk 2]
```diff
@@ -5,20 +5,30 @@
...
+Karpenter optimizes for scheduling latency and utilization efficiency using two complementary control loops:
 
-Karpenter optimizes for scheduling latency and utilization efficiency using two complementary control loops. First, is the allocator, a fast-acting latency-sensitive controller responsible for ensuring that incoming pods are scheduled as quickly as possible. Second, is the reallocator, a slow-acting cost-sensitive controller that replaces nodes as pods requests and capacity prices shift over time. Together, they maximize the availability and efficiency of your cluster.
+* The *allocator* is a fast-acting latency-sensitive controller that is responsible for ensuring that incoming pods are scheduled as quickly as possible.
+* The *reallocator* is a slow-acting cost-sensitive controller that replaces nodes as pod requests and capacity prices shift over time.
```
Comment (geoffcline) [inline README.md:29]: I didn't know about this breakdown, great info.

**Review by geoffcline — DISMISSED** [review]
:)

File: README.md (hunk 1 above)
Comment (ellistarn) [reply README.md:8]: I don't know if it does. Not rendering for me

**Review by ellistarn — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (chrisnegus) [reply README.md:8]: Oh darn. That's a Hugo/docsy thing. I forgot I'm not in the docs section. I'll fix it.

File: README.md (hunk 2 above)
Comment (chrisnegus) [reply README.md:29]: I didn't either. It was in the original README, so I left it.

**Review by geoffcline — APPROVED** [review]

**Pull request merged**